apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -1756,7 +1756,7 @@ export interface Extensions {
             };
         };
         emailProviders: {
-            configureEmailProvider: string;
+            configureEmailTemplates: string;
```

Comment:
why did we add this under email providers?

Reply:
This was discussed in the mail thread. Suggestion came from @jeradrutnam

Reply:
Where should this be go? As we discussed in the mail thread and offline, @jeradrutnam suggested to keep a link from Email Provider to email templates. For that, I add this i18n string.

Reply:
Yes @bhagyasakalanka that understanding is correct